Codigo/Protocol_Writes.bas

```diff
@@ -75,20 +75,14 @@ End Sub
 #End If
 ' \Begin: [Writes]
 
-Public Sub WriteConnected(ByVal UserIndex As Integer)
-        '<EhHeader>
+Public Function PrepareConnected()
         On Error GoTo WriteConnected_Err
-        '</EhHeader>
 100     Call Writer.WriteInt16(ServerPacketID.connected)
-102     Call modSendData.SendData(ToIndex, UserIndex)
```

Comment:
Don't we send the UserIndex? this is only sent upon successful login?

Reply:
we don't send user index to the client, it is internal for the server, the server maps a connection to every user index